packages/categorize/configure/src/defaults.js

```diff
@@ -5,6 +5,7 @@ export default {
     choicesLabel: '',
     lockChoiceOrder: true,
     removeTilesAfterPlacing: false,
+    allowMultiplePlacementsEnabled: 'Yes',
```

Comment:
Does this property have any other value other than "Yes" or "No"? If not, please make it a boolean value

Reply:
Yes, it have another one 'Set per Choice'.